airflow/__init__.py

```diff
@@ -74,11 +74,13 @@ def __getattr__(name):
 if not settings.LAZY_LOAD_PROVIDERS:
     from airflow import providers_manager
 
-    providers_manager.ProvidersManager().initialize_providers_manager()
+    manager = providers_manager.ProvidersManager()
+    manager.initialize_providers_list()
```

Comment:
```suggestion
```

Reply:
I think better to make it explicit here. It will work of course if we remove it but, I think this case is special as we know we want to initialize everything and it shows the reader our intent better.

Reply:
I disagree -- the intent here is to preload the hooks and links -- that we need to load the list to do that is to me an implementation detail that we don't strictly care about.

But I don't feel all that strongly about it.